### PR TITLE
Added 45 new welcome messages

### PR DESCRIPTION
**Pull request by sidtr0**
Thanks to all who gave their suggestions. It's very probable I might have made some formatting errors, so apologies in advance! 🙈

### CODE REVIEW

File: src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt [hunk 1]
```diff
@@ -32,7 +32,53 @@ object Messages {
...
+        "%name% has made the wise decision to hide behind a bush. However, %name% has also made the common mistake of hiding in the only bush in the area. This is how not to be seen.",
+        "May I have your attention, please?\nWill the real %name% please stand up?\nI repeat, will the real %name% please stand up?\nWe're gonna have a problem here",
+        "My name is %name%, Commander of the coders of the North, General of the IDE legions, loyal servant to the true emperor, Open Source. Father to an unfinished project, husband to an idea waiting to be started. And I will have my vengeance, in this life or the next.",
+        "git blame %name%",
+        "Oh, %name%, don't let's ask for the moon. We have the stars.",
+        "%name%, you can't fight in here! This is the War Room!",
+        "Open the pod bay doors, %name%.",
+        "That'll do, %name%. That'll do.",
+        "%name%, the question is\u0027nt how... but when...",
+        "%Name%, excited to time travel? Here, take this yellow raincoat to be known as the \u0027unchanged\u0027"
```
Comment (hbjydev) [inline src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt]:
```suggestion
        "%name%, excited to time travel? Here, take this yellow raincoat to be known as the \u0027unchanged\u0027"
```

File: src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt [hunk 2]
```diff
@@ -32,7 +32,53 @@ object Messages {
...
+        "Raise a glass to %name%!",
+        "A toast to %name%!",
+        "The world turned upside down, %name%!",
+        "The world will never be the same, %name%!",
+        "Uncaught Exception: TemplateRenderError(\"no user $name found in cache\")",
+        "Tell the story of tonight, %name%!",
+        "Stay alive, %name%!",
+        "History has its eyes on you, %name%!",
+        "The world\u0027s gonna know your name, %name%!",
+        "We are waiting in the wings for you, %main%!",
```
Comment (hbjydev) [inline src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt]:
```suggestion
        "We are waiting in the wings for you, %name%!",
```

**Review by hbjydev — CHANGES_REQUESTED** [review]

File: src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt (hunk 1 above)
Comment (sidtr0) [reply src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt]: Thanks!

File: src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt (hunk 2 above)
Comment (sidtr0) [reply src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt]: Oh boy, that was embarrassing 🙈 
Thanks!

File: src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt [hunk 3]
```diff
@@ -32,7 +32,53 @@ object Messages {
...
-        "Here\u0027s your sword, shield, helmet, now go %name%, and vanquish those bugs!"
+        "Here\u0027s your sword, shield, helmet, now go %name%, and vanquish those bugs!",
+        "%name%, I\u0027ve got a feeling we're not in Kansas anymore...",
+        "Don\u0027t throw away your shot, %name%!",
+        "One, two, three, four, five, six, seven, eight, nine. It\u0027s %name%!",
+        "Raise a glass to %name%!",
+        "A toast to %name%!",
+        "The world turned upside down, %name%!",
+        "The world will never be the same, %name%!",
+        "Uncaught Exception: TemplateRenderError(\"no user $name found in cache\")",
```
Comment (hbjydev) [inline src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt]:
```suggestion
        "Uncaught Exception: TemplateRenderError(\"no user %name% found in cache\")",
```

**Review by hbjydev — CHANGES_REQUESTED** [review]

File: src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt (hunk 3 above)
Comment (ddivad195) [inline src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt]: I think this is failing the build. Maybe it should be "%name%" as well?

Comment (hbjydev) [reply src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt]: Just reviewed that lol @ddivad195

Comment (sidtr0) [reply src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt]: Indeed. Thanks a lot :D

**Comment by hbjydev** [comment]
lgtm! :)

**Pull request merged**